docs/paper/dev/getting-started/project-setup.md

```diff
@@ -0,0 +1,123 @@
+---
+slug: /dev/project-setup
+---
+
+# Paper Project Setup
+
+As the Paper team primarily uses IntelliJ IDEA, this guide will be focused on that IDE. 
+However, the steps below should be applicable to other IDEs as well, with some minor changes.
+
+
+### Creating a new project
+
+Open your IDE and select the option to create a new project. 
+In Intellij, you will get the option to select the type of project you want to create. 
+Select `Gradle - Kotlin DSL` and click `Create`.
+
+You will land into the `build.gradle.kts` file where you can add your dependencies.
+
+### Adding Paper as a dependency
+
+To add Paper as a dependency, you will need to add the Paper repository to your `build.gradle.kts` file as well as the dependency itself.
+
+```kotlin
+repositories {
+    mavenCentral()
+    maven {
+        url = uri("https://repo.papermc.io/repository/maven-public/")
+    }
+}
+
+dependencies {
+    compileOnly("io.papermc.paper:paper-api:1.19.2-R0.1-SNAPSHOT")
+}
+
+java {
+    toolchain.languageVersion.set(JavaLanguageVersion.of(17))
+}
+```
+
+:::info
+
+To see the versions of Paper you want to use, visit our [Downloads](https://www.papermc.io/downloads) page.
+
+:::
+
+### Setting up the `src` directory
+
+To set up the `src` directory, you will need to create a new directory called `src` and then create a new directory called `main` inside of it.
+Inside of `main`, create two new directories called `java` and `resources`.
```

Comment:
I believe Intellij does this automatically?

Reply:
If you create an empty project, it won't. A new project will, though- but it takes it 30+ seconds to set up.

Reply:
So is this a "this is fine as it is", or a "needs removing"? I could always just add a "If your project doesn't have these folders, add them like this - otherwise skip."

Reply:
I think it's fine as is, as it is something some people will encounter. I might just add, like you said, making it clearly optional/only if it does not already exist.